structure.md

```diff
@@ -0,0 +1,29 @@
+# Structure
+
+Having a good structure is essential for our culture. We derive and maintain our structure from the learnings of the open-source community to maximize the odds to [better decisions](decision-making.md), innovation, and sustainability.
+
+Presently, our structure constitutes of the following supporting natures.
+
+## Flat
+
+The flat structure encourages members, regardless of who they are, to have an equal opportunity to contribute to the projects that suit their passions and interests. Such a structure plays a vital role in how we make decisions, collaborate, and communicate.
+
+Within the flat structure, members are naturally encouraged to contribute with minimal supervision. However, this also means that members are encouraged to provide support, guidance, and mentorship to other members. Specifically for new members, existing members would support with a more tailored guide and progressively transition them to be more independent and proactive to contribute creative solutions or even new problems that align with the business objectives and strategies.
+
+## Fluid
+
+To synergize with the flat nature and maximize innovation and sustainability of the organization and projects, we maintain a fluid structure that empowers members to work on projects relevant to their interests and passions. The fluidity profoundly enhances the productivity of the organization and projects while sustaining members with less burnout.
+
+However, balancing existing responsibility and flexibility to move around the organization and projects is also essential. We encourage members to proactively attract other members toward their existing projects and gradually impart more experience and knowledge. To achieve so, we leverage [good communication and collaboration practices](communications/README.md) and invite other members to peer-review progress to observe, learn, and contribute, usually in small and discrete scopes to not overwhelm.
+
+At the same time, we encourage members to proactively explore and contribute to other projects in similar fashions through observation and providing peer-reviews. Such practice would enable a streamlined and progressive process to transfer knowledge and domain expertise, minimize the need for a lumpsum knowledge transfer process, and mitigate the risk of lack of thorough documentation.
+
+## Distributed
+
+We share a distributed workplace. Where members work depends on their individual needs and preferences, which offers the following key advantages:
```

Comment:
Where a member's work depends on their individual needs and preferences and which offers the following advantages: